<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceViewPanel</class>
 <widget class="QWidget" name="SliceViewPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="sliceToolbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageDefault">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnZoomToFit">
         <property name="font">
          <font>
           <family>Helvetica</family>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>zoom to fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSliceInfo">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePolygon"/>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QScrollBar" name="inSlicePosition">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="GenericSliceView" name="sliceView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <widget class="CrosshairsInteractionMode" name="imCrosshairs" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>293</height>
       </rect>
      </property>
     </widget>
     <widget class="CrosshairsInteractionMode" name="imZoomPan" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>293</height>
       </rect>
      </property>
     </widget>
     <widget class="ThumbnailInteractionMode" name="imThumbnail" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>293</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericSliceView</class>
   <extends>QWidget</extends>
   <header>GenericSliceView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrosshairsInteractionMode</class>
   <extends>QWidget</extends>
   <header>CrosshairsInteractionMode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThumbnailInteractionMode</class>
   <extends>QWidget</extends>
   <header>ThumbnailInteractionMode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
